sklearn/svm/classes.py

```diff
@@ -620,7 +629,16 @@ class NuSVC(BaseSVC):
 
     gamma : float, optional (default='auto')
         Kernel coefficient for 'rbf', 'poly' and 'sigmoid'.
-        If gamma is 'auto' then 1/n_features will be used instead.
+
+        Current default is 'auto' which uses 1 / n_features,
+        if gamma='scale' is passed then it uses 1 / (n_features * X.std())
+        features. The current default of gamma, 'auto', will change to 'scale'
+        in version 0.22. 'auto_deprecated', a deprecated version of 'auto' is
```

Comment:
I've documented 'auto_deprecated' here. Does this look okay (place and wording)?